x-pack/plugins/uptime/public/components/overview/alerts/alert_tls.tsx

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiExpression,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiLink,
+  EuiPopover,
+  EuiSpacer,
+} from '@elastic/eui';
+import React, { useState } from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { useKibana } from '../../../../../../../src/plugins/kibana_react/public';
+import { TlsTranslations } from './translations';
+
+interface SettingsMessageExpressionPopoverProps {
+  'aria-label': string;
+  description: string;
+  id: string;
+  setAlertFlyoutVisible: (value: boolean) => void;
+  value: string;
+}
+
+const SettingsMessageExpressionPopover: React.FC<SettingsMessageExpressionPopoverProps> = ({
+  'aria-label': ariaLabel,
+  description,
+  setAlertFlyoutVisible,
+  value,
+  id,
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const kibana = useKibana();
+  const href = kibana.services?.application?.getUrlForApp('uptime#/settings');
+  return (
+    <EuiPopover
+      id={id}
+      anchorPosition="downLeft"
+      button={
+        <EuiExpression
+          aria-label={ariaLabel}
+          color="secondary"
+          description={description}
+          isActive={isOpen}
+          onClick={() => setIsOpen(!isOpen)}
+          value={value}
+        />
+      }
+      isOpen={isOpen}
+      closePopover={() => setIsOpen(false)}
+    >
+      <FormattedMessage
+        id="xpack.uptime.alerts.tls.settingsPageNav.text"
+        defaultMessage="You can edit these thresholds on the {settingsPageLink}."
+        values={{
+          settingsPageLink: (
+            // this link is wrapped around a span so we can also change the UI state
+            // and hide the alert flyout before triggering the navigation to the settings page
+            <EuiLink href={href}>
+              <span
+                onClick={() => {
+                  setAlertFlyoutVisible(false);
+                }}
+                onKeyUp={e => {
+                  if (e.key === 'Enter') {
+                    setAlertFlyoutVisible(false);
+                  }
+                }}
+              >
+                settings page
+              </span>
+            </EuiLink>
+          ),
+        }}
+      />
+    </EuiPopover>
+  );
```

Comment:
This is nicely done, but now that i see it in action in UI, it doesn't makes a good experience of user, looks bit awkward. But i think we can live with this, until we find a better solution

Reply:
I would've preferred being able to edit the settings directly, but the spec wanted them to go to the settings page. I agree it's a bit janky, but it also increases visibility of the dynamic settings feature.